src/NuGet.Clients/PackageManagement.PowerShellCmdlets/Cmdlets/GetPackageCommand.cs

```diff
@@ -359,5 +361,10 @@ private int AskToContinueDisplayPackages()
             WriteLine();
             return choice;
         }
+
+        private static bool IsAutoReferenced(PackageReference reference)
+        {
+            return (reference as BuildIntegratedPackageReference)?.Dependency?.AutoReferenced == true;
```

Comment:
this returns a ``bool?`` since it uses ``?``, ``== true`` converts ``bool? -> bool``